dependencies.list

```diff
@@ -1,5 +1,5 @@
 PACKAGE_NAME=realm-js
 VERSION=2.26.0
-REALM_CORE_VERSION=5.14.0
-REALM_SYNC_VERSION=3.15.0
-REALM_OBJECT_SERVER_VERSION=3.17.1
+REALM_CORE_VERSION=5.19.1
+REALM_SYNC_VERSION=4.4.2
```

Comment:
Changelog needs to be updated with the changes from these core and sync versions.